### PR TITLE
e2e: add log for running which cleanup action

### PR DESCRIPTION
**Pull request by pacoxu**
#### What type of PR is this?
/kind cleanup
/sig testing

This PR only tries to add some log for cleanup actions.
/cc jayunit100
is this enough? 

#### Which issue(s) this PR fixes:
xref #101870

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

### CODE REVIEW

File: test/e2e/framework/cleanup.go [hunk 1]
```diff
@@ -70,6 +73,7 @@ func RunCleanupActions() {
 	}()
 	// Run unlocked.
 	for _, fn := range list {
+		e2elog.Logf("Run Cleanup Action: %v", runtime.FuncForPC(reflect.ValueOf(fn).Pointer()).Name())
```
Comment (neolit123) [inline test/e2e/framework/cleanup.go:76]:
```suggestion
		e2elog.Logf("Running cleanup action: %v", runtime.FuncForPC(reflect.ValueOf(fn).Pointer()).Name())
```

the change looks fine in general, but would it spam the logs a lot?

/approve

Comment (pacoxu) [reply test/e2e/framework/cleanup.go]: We need to make a new choice
- no spamming logs
- know which action hang

**Pull request merged**